commit 323d397747ff81a5706cce63f45cc1b2109db252
Author: mjw <mjw@a5019735-40e9-0310-863c-91ae7b9d1cf9>
Date:   Thu Dec 12 21:20:48 2013 +0000

    Bug 328711 valgrind.1 manpage "memcheck options" section is badly generated
    
    Add missing para tags inside listitems. (Miroslav Franc, mfranc@redhat.com)
    
    git-svn-id: svn://svn.valgrind.org/valgrind/trunk@13758 a5019735-40e9-0310-863c-91ae7b9d1cf9

diff --git a/memcheck/docs/mc-manual.xml b/memcheck/docs/mc-manual.xml
index a53bf86..e913e89 100644
--- a/memcheck/docs/mc-manual.xml
+++ b/memcheck/docs/mc-manual.xml
@@ -610,14 +610,14 @@ when <option>--leak-check=full</option> is specified. </para>
 in one of the following ways:
 
 <itemizedlist>
-  <listitem>a comma separated list of one or more of
-    <option>definite indirect possible reachable</option>.
+  <listitem><para>a comma separated list of one or more of
+    <option>definite indirect possible reachable</option>.</para>
   </listitem>
 
-  <listitem><option>all</option> to specify the complete set (all leak kinds).
+  <listitem><para><option>all</option> to specify the complete set (all leak kinds).</para>
   </listitem>
 
-  <listitem><option>none</option> for the empty set.
+  <listitem><para><option>none</option> for the empty set.</para>
   </listitem>
 </itemizedlist>
 
@@ -721,16 +721,16 @@ is <option>--errors-for-leak-kinds=definite,possible</option>
       one of the following ways:
 
         <itemizedlist>
-          <listitem>a comma separated list of one or more of
-            <option>definite indirect possible reachable</option>.
+          <listitem><para>a comma separated list of one or more of
+            <option>definite indirect possible reachable</option>.</para>
           </listitem>
           
-          <listitem><option>all</option> to specify the complete set (all leak kinds).
+          <listitem><para><option>all</option> to specify the complete set (all leak kinds).
             It is equivalent to
-            <option>--show-leak-kinds=definite,indirect,possible,reachable</option>.
+            <option>--show-leak-kinds=definite,indirect,possible,reachable</option>.</para>
           </listitem>
           
-          <listitem><option>none</option> for the empty set.
+          <listitem><para><option>none</option> for the empty set.</para>
           </listitem>
         </itemizedlist>
       </para>
@@ -763,17 +763,17 @@ is <option>--errors-for-leak-kinds=definite,possible</option>
         The heuristic set is specified in one of the following ways:
 
         <itemizedlist>
-          <listitem>a comma separated list of one or more of
-            <option>stdstring newarray multipleinheritance</option>.
+          <listitem><para>a comma separated list of one or more of
+            <option>stdstring newarray multipleinheritance</option>.</para>
           </listitem>
           
-          <listitem><option>all</option> to activate the complete set of
+          <listitem><para><option>all</option> to activate the complete set of
             heuristics.
             It is equivalent to
-            <option>--leak-check-heuristics=stdstring,newarray,multipleinheritance</option>.
+            <option>--leak-check-heuristics=stdstring,newarray,multipleinheritance</option>.</para>
           </listitem>
           
-          <listitem><option>none</option> for the empty set.
+          <listitem><para><option>none</option> for the empty set.</para>
           </listitem>
         </itemizedlist>
       </para>
@@ -797,18 +797,24 @@ is <option>--errors-for-leak-kinds=definite,possible</option>
       <para>These options provide an alternative way to specify the leak kinds to show:
         <itemizedlist>
           <listitem>
+	    <para>
             <option>--show-reachable=no --show-possibly-lost=yes</option> is equivalent to
             <option>--show-leak-kinds=definite,possible</option>.
+	    </para>
           </listitem>
           <listitem>
+	    <para>
             <option>--show-reachable=no --show-possibly-lost=no</option> is equivalent to
             <option>--show-leak-kinds=definite</option>.
+	    </para>
           </listitem>
           <listitem>
+	    <para>
             <option>--show-reachable=yes</option> is equivalent to
             <option>--show-leak-kinds=all</option>.
             Note that  <option>--show-possibly-lost=no</option> has no effect
             if <option>--show-reachable=yes</option> is specified.
+	    </para>
           </listitem>
         </itemizedlist>
       </para>
--- valgrind-3.9.0/docs/valgrind.1.orig	2013-12-12 22:34:11.912168437 +0100
+++ valgrind-3.9.0/docs/valgrind.1	2013-12-12 22:34:18.686421525 +0100
@@ -2,12 +2,12 @@
 .\"     Title: VALGRIND
 .\"    Author: [see the "Author" section]
 .\" Generator: DocBook XSL Stylesheets v1.78.1 <http://docbook.sf.net/>
-.\"      Date: 11/01/2013
+.\"      Date: 12/12/2013
 .\"    Manual: Release 3.9.0
 .\"    Source: Release 3.9.0
 .\"  Language: English
 .\"
-.TH "VALGRIND" "1" "11/01/2013" "Release 3.9.0" "Release 3.9.0"
+.TH "VALGRIND" "1" "12/12/2013" "Release 3.9.0" "Release 3.9.0"
 .\" -----------------------------------------------------------------
 .\" * Define some portability stuff
 .\" -----------------------------------------------------------------
@@ -1080,8 +1080,8 @@
 .IP \(bu 2.3
 .\}
 a comma separated list of one or more of
-            \fBdefinite indirect possible reachable\fR\&.
-          .RE
+\fBdefinite indirect possible reachable\fR\&.
+.RE
 .sp
 .RS 4
 .ie n \{\
@@ -1091,10 +1091,10 @@
 .sp -1
 .IP \(bu 2.3
 .\}
-\fBall\fR to specify the complete set (all leak kinds)\&.
-            It is equivalent to
-            \fB\-\-show\-leak\-kinds=definite,indirect,possible,reachable\fR\&.
-          .RE
+\fBall\fR
+to specify the complete set (all leak kinds)\&. It is equivalent to
+\fB\-\-show\-leak\-kinds=definite,indirect,possible,reachable\fR\&.
+.RE
 .sp
 .RS 4
 .ie n \{\
@@ -1104,8 +1104,9 @@
 .sp -1
 .IP \(bu 2.3
 .\}
-\fBnone\fR for the empty set\&.
-          .RE
+\fBnone\fR
+for the empty set\&.
+.RE
 .sp
 .RE
 .PP
@@ -1130,8 +1131,8 @@
 .IP \(bu 2.3
 .\}
 a comma separated list of one or more of
-            \fBstdstring newarray multipleinheritance\fR\&.
-          .RE
+\fBstdstring newarray multipleinheritance\fR\&.
+.RE
 .sp
 .RS 4
 .ie n \{\
@@ -1141,11 +1142,10 @@
 .sp -1
 .IP \(bu 2.3
 .\}
-\fBall\fR to activate the complete set of
-            heuristics\&.
-            It is equivalent to
-            \fB\-\-leak\-check\-heuristics=stdstring,newarray,multipleinheritance\fR\&.
-          .RE
+\fBall\fR
+to activate the complete set of heuristics\&. It is equivalent to
+\fB\-\-leak\-check\-heuristics=stdstring,newarray,multipleinheritance\fR\&.
+.RE
 .sp
 .RS 4
 .ie n \{\
@@ -1155,8 +1155,9 @@
 .sp -1
 .IP \(bu 2.3
 .\}
-\fBnone\fR for the empty set\&.
-          .RE
+\fBnone\fR
+for the empty set\&.
+.RE
 .sp
 Note that these heuristics are dependent on the layout of the objects produced by the C++ compiler\&. They have been tested with some gcc versions (e\&.g\&. 4\&.4 and 4\&.7)\&. They might not work properly with other C++ compilers\&.
 .RE
@@ -1173,9 +1174,10 @@
 .sp -1
 .IP \(bu 2.3
 .\}
-\fB\-\-show\-reachable=no \-\-show\-possibly\-lost=yes\fR is equivalent to
-            \fB\-\-show\-leak\-kinds=definite,possible\fR\&.
-          .RE
+\fB\-\-show\-reachable=no \-\-show\-possibly\-lost=yes\fR
+is equivalent to
+\fB\-\-show\-leak\-kinds=definite,possible\fR\&.
+.RE
 .sp
 .RS 4
 .ie n \{\
@@ -1185,9 +1187,10 @@
 .sp -1
 .IP \(bu 2.3
 .\}
-\fB\-\-show\-reachable=no \-\-show\-possibly\-lost=no\fR is equivalent to
-            \fB\-\-show\-leak\-kinds=definite\fR\&.
-          .RE
+\fB\-\-show\-reachable=no \-\-show\-possibly\-lost=no\fR
+is equivalent to
+\fB\-\-show\-leak\-kinds=definite\fR\&.
+.RE
 .sp
 .RS 4
 .ie n \{\
@@ -1197,11 +1200,14 @@
 .sp -1
 .IP \(bu 2.3
 .\}
-\fB\-\-show\-reachable=yes\fR is equivalent to
-            \fB\-\-show\-leak\-kinds=all\fR\&.
-            Note that  \fB\-\-show\-possibly\-lost=no\fR has no effect
-            if \fB\-\-show\-reachable=yes\fR is specified\&.
-          .RE
+\fB\-\-show\-reachable=yes\fR
+is equivalent to
+\fB\-\-show\-leak\-kinds=all\fR\&. Note that
+\fB\-\-show\-possibly\-lost=no\fR
+has no effect if
+\fB\-\-show\-reachable=yes\fR
+is specified\&.
+.RE
 .sp
 .RE
 .PP
